src/scicookie/{{cookiecutter.project_slug}}/docs-sphinx/readme.rst

```diff
@@ -1 +1,4 @@
+Introduction
```

Comment:
I think that now I got the point you mentioned on discord.
that makes more sense to call this file to intro.rst
also please rename it  https://github.com/osl-incubator/scicookie/pull/124/files#diff-e5704837157913d9457988b9450bc2f363e27663bd5e2ecd098e580a44e9adb9R11